### ISSUE TITLE
Absolute positioning of legends when "orient" is "none"

### ISSUE THREAD

**Issue by jwoLondon** [vega/vega-lite#4923]
The [legend documentation](https://vega.github.io/vega-lite/docs/legend.html#properties) (and schema) suggests that one option for legend orient is `none`. Presumably this reflects the [similar option in Vega](https://vega.github.io/vega/docs/legends/#orientation). In Vega this allows a legend to be positioned with absolute coordinates, such as the following:

```javascript
"legends": [
    {
      "title": "Weight",
      "opacity": "oScale",
      "symbolType": "circle",
      "orient": "none",
      "encode": {
        "legend": {
            "enter": {
              "x":{ "value":320},
              "y": {"value":20}
            }
          }
      }    
    }
  ]
```

But as far as I can see, there is no way to do the equivalent of this in Vega-Lite. It would be useful if we could, perhaps something like this:

```javascript
"legend": {"orient": "none", "x": 320, "y": 20}
```

While for simple charts, absolute legend positioning is unlikely to be critical, for map-based legends this requirement is much more common where a legend needs to be positioned in the map space but not occluding important map features.

**Comment by jheer** [comment]
I’ve already added (but not yet released) top level legendX/Y properties in Vega. Once released those can propagate to VL.

**Comment by jwoLondon** [comment]
Closed following release of VL 3.3 with `legendX` and `legendY` properties.
